Runtime/Scripts/Timeline/GLTFRecorder.cs

```diff
@@ -175,7 +175,48 @@ internal void endRecording(out Dictionary<Transform, AnimationData>? param)
 			param = recordingAnimatedTransforms;
 		}
 
-		public void EndRecording(string filename, string sceneName = "scene", GLTFSettings? settings = null)
+		public bool TryEndRecordingDirect()
```

Comment:
I feel like my original comment about this method still holds true?
If you disagree about my opinion, lets talk about it

Reply:
I kinda disagree/ dont see a better naming here. Lets talk